core/scripts/local/DeployEMP.js

```diff
@@ -62,7 +62,7 @@ const deployEMP = async callback => {
 
     // Create a new EMP
     const constructorParams = {
-      expirationTimestamp: (await expiringMultiPartyCreator.VALID_EXPIRATION_TIMESTAMPS(6)).toString(),
+      expirationTimestamp: "1601510400", // 2020-09-01T00:00:00.000Z
```

Comment:
FYI this will work until september 1st 2020. To be safe you should add some amount of fixed time to `Math.round(Date.now()/1000)`

Reply:
Or just add a comment that this is expected to break then.